src/Gir.Tests/ImportantGeneratables/GObject.cs

```diff
@@ -23,7 +23,7 @@ public Object (UIntPtr object_type, string first_property_name, ... var_args) :
 		{
 		}
 
-		static extern IntPtr g_object_newv (UIntPtr object_type, uint n_parameters, Parameter parameters);
+		static extern Object g_object_newv (UIntPtr object_type, uint n_parameters, Parameter parameters);
```

Comment:
This doesn't look right. We don't want the pinvoke signature to use the C# type.

Reply:
True, but Classes use RetVal.CSharpType right now which points to Object

Reply:
I think the test changed because Object was not resolved correctly?

Reply:
Aaaah, yes! I thought we used CType, but we don't 👍